Odata-docs/changelog/odatalib-7x.md

```diff
@@ -28,6 +28,14 @@ There was a lot of vestigial code left around from the OData v1-3 days that we
 
 Most of our API improvements fall into the category of namespace simplifications or updating verbiage. The single most impactful change that we made was deciding to merge entity type and complex type in ODataLib. We did this because complex type and entity type are becoming more and more similar in the protocol, but we continue to pay overhead to make things work for both of them.
 
+## ODataLib 7.14.0 Release
+
+***Fixed Bugs***
+
+* fix: ExpressionWriter only prepend slash to method when we have a parent by @uffelauesen in [#2572](https://github.com/OData/odata.net/pull/2572)
+* Ported DataServiceContext.AutoNullPropagation from V3 ODL by @uffelauesen in [#2573](https://github.com/OData/odata.net/pull/2573)
+* IncrementVersion.ps1 fixes by @corranrogue9 in [#2581](https://github.com/OData/odata.net/pull/2581)
```

Comment:
don't need this in the release note.

Reply:
To clarify, I don't need the third bullet point?

Reply:
Yes, I think we don't need list "increment version" changes in the release note.